src/server/services/duplicatesService.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+
+const getDuplicates = require('./helpers/getDuplicates');
+
+const inputFilePath = "src/inputData/";
+const outputFilePath = "src/outputData/";
+
+const indexFilePath = "./src/dataBase/file-indexData.json";
```

Comment:
`file-indexData` naming looks strange.
change to `file-index-data` or `fileIndexData`